komapper-codegen/src/main/java/org/komapper/codegen/CodeGenerator.java

```diff
@@ -69,7 +69,8 @@ public void generateEntities(
       p.println();
       var className = classNameResolver.resolve(table);
       if (useSelfMapping) {
-        p.println("@KomapperEntity");
+        p.println(
+            "@KomapperEntity([\"" + StringUtil.snakeToLowerCamelCase(table.getName()) + "\"])");
```

Comment:
Can you tell me why you explicitly specify `KomapperEntity.aliases`?
And why do you use `table.getName()`?

If not specified, the name is resolved from the entity class at KSP execution.

Reply:
@nakamura-to 
Thanks for feedback.

My idea was to make the query DSL more natural.

```
QueryDsl.from(Meta.addresses).where { Meta.addresses.addressId eq 1 }
```

If the entity name was singularized from plural, it will be generated with a singular name unless an alias is set.
For example, `Meta.address`.

However, if you are not converting to singular, then there is no gap between the entity name and the table name, so this is certainly an unnecessary setting. I can modified to only output an alias when `singularize` is `true`.

Reply:
Thank you for explaining it to me.

>  I can modified to only output an alias when singularize is true.

That is a good idea.